ppl/zqd/recruiter/recruiter.go

```diff
@@ -0,0 +1,178 @@
+package recruiter
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"sync"
+	"time"
+)
+
+// WorkerPool is the internal state of the recruiter system
+// The pools are exported only for use in unit tests
```

Comment:
Doc comments should comprise complete sentences. See https://golang.org/doc/effective_go.html#commentary.

And this says the pools are exported, but they aren't.

Reply:
Thanks! I will correct that in my next PR.